code/game/objects/effects/overlays.dm

```diff
@@ -485,6 +485,31 @@
 	duration = 8
 	randomdir = 0
 
+/obj/effect/overlay/telesci
```

Comment:
why the fuck did you not use `/obj/effect/overlay/temp`, holy shit, you're in the same file as it